packages/cli/src/util/input/select-org.ts

```diff
@@ -36,10 +36,12 @@ export default async function selectOrg(
 
   const choices: Choice[] = [
     ...personalAccountChoice,
-    ...teams.map<Choice>(team => ({
-      name: team.name || team.slug,
-      value: { type: 'team', id: team.id, slug: team.slug },
-    })),
+    ...teams
+      .sort(a => (a.id === user.defaultTeamId ? -1 : 1))
```

Comment:
This sorting function is a bit strange to me (I've never seen a comparator which does not operate on the `b` parameter). Also, it's not clear to me what the fallback behavior is (for the non-`defaultTeamId` case), but we should probably make an explicit decision about that (i.e. sort by `slug`).

Reply:
Yeah, unfortunately all of the ways to say "sort a specific item to the top of the array but leave everything else untouched" in JavaScript are weird, with this being the least weird. I did test it on every case (item at start, item at end, item inbetween) and it works fine -- `-1`s get sorted earlier, `1`s get sorted later, equal neighbors get left untouched. 

As for fallback behavior, there is none today and there also is none after this PR merges -- we just display your teams in the order they come back from `getTeams(client)`.